go.mod

```diff
@@ -430,6 +431,9 @@ replace (
 	gopkg.in/couchbaselabs/gocbconnstr.v1 => github.com/couchbaselabs/gocbconnstr v1.0.5
 )
 
+// TODO: Remove this when upgrading the library to 2.0.10 or higher
```

Comment:
If I didn't add this replace, `go mod tidy` would always bring it back to 2.0.9. No idea why